docs/ENVIRONMENT.md

```diff
@@ -294,6 +294,9 @@ All tools:
   first existing unit listed in the environment variable, and
   `timedatectl set-ntp off` disables and stops all listed units.
 
+* `$SYSTEMD_ETC_ADJTIME` - override the path to the hardware clock setting.
```

Comment:
"settings file"? Here and below, I think with "file" the descriptions would sound much better.